src/Microsoft.AspNetCore.Mvc.Razor.Precompilation.Design/Internal/FIleProviderUtilities.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.AspNetCore.Mvc.Razor.Compilation;
+using Microsoft.Extensions.FileProviders;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.Precompilation.Design.Internal
+{
+    public static class FileProviderUtilities
```

Comment:
Why does this need its own class? can't these methods be inlined in the one class that uses this?